Makefile

```diff
@@ -55,6 +55,7 @@ BUILDTAGS ?= \
 	$(shell hack/btrfs_tag.sh) \
 	$(shell hack/systemd_tag.sh) \
 	$(shell hack/libsubid_tag.sh) \
+	${EXTRA_BUILD_TAGS} \
```

Comment:
or use the `override` directive explicitly

Reply:
Idea SGTM but the inconsistency between no-underscore pre-existing `BUILDTAGS` and the new underscored `BUILD_TAGS` is going to cause headaches. Would you consider removing the second underscore, making it just `EXTRA_BUILDTAGS`? And please include a nearby comment explaining that this is intended for command-line overrides; that will save a future maintainer the wasted time of looking for where it's defined.

Reply:
I have opted for not using `BUILDFLAGS += ` as now a user can override the build flags completely and with `+=` it wouldn't be that simple

Reply:
Even with `EXTRA_BUILDTAGS`, we're appending to `BUILDTAGS` or am I missing something? The behavior seems to be the same.

Reply:
@danishprakash I don't understand your question, and you're commenting on an obsolete diff. Could you please  look at the current diffs and then, if you still have a question, ask it on the current diff, and perhaps include an at-mention of the person you're asking the question of? Thank you.

Reply:
Sure, I was proposing we use `+=` to declare the original `BUILDTAGS` variable instead of adding a second `EXTRA_BUILDTAGS` variable. But @dcermak clarified that if you don't want the existing tags defined in the Makefile to be used for some reason, it won't be possible with `+=`. So we're good here. Sorry for the confusion!